components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonUtils.java

```diff
@@ -182,4 +191,49 @@ public static String getUserConsumerId() {
         return consumerId;
     }
 
+    public static void addAdminGroup(int tenantId) throws UserStoreException {
+        try {
+            UserStoreManager userStoreManager = (UserStoreManager) SCIMCommonComponentHolder.getRealmService().
+                    getTenantUserRealm(tenantId).getUserStoreManager();
+
+            if (userStoreManager.isSCIMEnabled()) {
+
+                SCIMGroupHandler scimGroupHandler = new SCIMGroupHandler(userStoreManager.getTenantId());
+
+                String domainName = UserCoreUtil.getDomainName(userStoreManager.getRealmConfiguration());
+                if (domainName == null) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Domain name is null and setting default domain as "
+                                + UserCoreConstants.PRIMARY_DEFAULT_DOMAIN_NAME);
+                    }
+                    domainName = UserCoreConstants.PRIMARY_DEFAULT_DOMAIN_NAME;
```

Comment:
Follow the same suggestions given in https://github.com/wso2-extensions/identity-inbound-provisioning-scim/pull/49